Sources/StructuredQueriesMacros/TableMacro.swift

```diff
@@ -1038,7 +1039,7 @@ extension TableMacro: MemberMacro {
       [\(writableColumns.map { "QueryValue.columns.\($0)" as ExprSyntax }, separator: ", ")]
       }
       public var queryFragment: QueryFragment {
-      "\(selectedColumns.map { #"\(self.\#($0))"# as ExprSyntax }, separator: ", ")"
+      "\(raw: selectedColumns.map { #"\(self.\#($0))"# }.joined(separator: ", "))"
```

Comment:
Running tests I noticed a lot of warnings from SwiftSyntax parsing. These small changes address them.